docker-push/action.yml

```diff
@@ -24,13 +21,14 @@ inputs:
     required: true
   deployment_env:
```

Comment:
We might want to remove `deployment_env` from the next version. 

If we do end up creating deployment events on application repositories, we should do it from Argo instead.